java/test/jmri/jmrix/marklin/MarklinThrottleManagerTest.java

```diff
@@ -9,30 +9,26 @@
  */
 public class MarklinThrottleManagerTest extends jmri.managers.AbstractThrottleManagerTestBase {
 
-    private MarklinTrafficController tc;
-    private MarklinSystemConnectionMemo memo;
+    private MarklinTrafficController tc = null;
+    private MarklinSystemConnectionMemo memo = null;
```

Comment:
No need to change, but Java automatically sets the fields to null.

The exception is a final field that's not assigned in the constructor. In that case, Java requires the value to be explicit set. Probably because a final field almost never have the value null.